X/Xorg_libX11/build_tarballs.jl

```diff
@@ -3,23 +3,20 @@
 using BinaryBuilder
 
 name = "Xorg_libX11"
-version = v"1.8.6"
+version = v"1.8.12"
 
 # Collection of sources required to build libX11
 sources = [
     ArchiveSource("https://www.x.org/archive/individual/lib/libX11-$(version).tar.xz",
-                  "59535b7cc6989ba806a022f7e8533b28c4397b9d86e9d07b6df0c0703fa25cc9"),
+                  "fa026f9bb0124f4d6c808f9aef4057aad65e7b35d8ff43951cef0abe06bb9a9a"),
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
-cd $WORKSPACE/srcdir/libX11-*/
-CPPFLAGS="-I${prefix}/include"
-# When compiling for things like ppc64le, we need newer `config.sub` files
-update_configure_scripts
-./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target} --enable-malloc0returnsnull=no
-# For some obscure reason, this Makefile may not get the value of CPPFLAGS
-sed -i "s?CPPFLAGS = ?CPPFLAGS = ${CPPFLAGS}?" src/util/Makefile
+cd $WORKSPACE/srcdir/libX11-*
+./configure --prefix=${prefix} --build=${MACHTYPE} --host=${target}
+# # For some obscure reason, this Makefile may not get the value of CPPFLAGS
+# sed -i "s?CPPFLAGS = ?CPPFLAGS = ${CPPFLAGS}?" src/util/Makefile
```

Comment:
If this is not needed can it be removed? Or have a comment to explain why it's currently not needed..

Reply:
Removed.